lib/tls.js

```diff
@@ -15,9 +15,9 @@ exports.SLAB_BUFFER_SIZE = 10 * 1024 * 1024;
 
 exports.DEFAULT_CIPHERS =
     // TLS 1.2
-    'ECDHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA256:AES128-GCM-SHA256:' +
+    'ECDHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA256:' +
```

Comment:
@shigeki The discussion got split a bit here. see https://github.com/iojs/io.js/issues/818#issuecomment-74237599 for my latest test with Fedor's list.